src/components/Authentication.jsx

```diff
@@ -50,7 +51,7 @@ export function AuthProvider({ children }) {
     verb: 'POST',
   });
 
-  const { refetch, loading } = useGet('/api/account/users/me', { lazy: true });
+  const { mutate: getMe, loading } = useMutate({ path: '/api/account/users/me', verb: 'GET' });
```

Comment:
is there a reason why it's a mutation? it's just a get request and the lazy would give it a similar effect

Reply:
I couldn't get it working. `refetch` didn't return any object back when I was debugging through vscode. When using `{ data, refetch, loading }` instead, `data` was not updated when `refetch` is called.

I think the intention is there, just the execution looks wrong here due to the syntax (ofc both work the same way as discussed here - https://github.com/contiamo/restful-react/issues/227).